roles/StackStorm.st2smoketests/tasks/st2chatops.yml

```diff
@@ -1,15 +1,41 @@
 ---
+- name: Prepare temp file to track retry attempts
```

Comment:
maybe at the top of this file, some comments in a summary of how this retry functionality works would be useful (what the temporary file is for, etc)